tests/test_other.py

```diff
@@ -790,10 +790,14 @@ def test_cmake_install(self):
     self.run_process(['cmake', '--build', 'build2', '--target', 'install'])
 
   def test_cmake_find_sdl2(self):
-    os.mkdir('build')
-    self.run_process([EMCMAKE, 'cmake', test_file('cmake/find_sdl2')], cwd='build')
-    self.run_process(['cmake', '--build', 'build'])
-    self.assertContained('SDL version: 2.0.', self.run_js('build/sdl2.js'))
+    self.run_process([EMCMAKE, 'cmake', test_file('cmake/find_sdl2')])
```

Comment:
would it make sense to just roll the find_sdl2 test into find_modules? e.g. why have OpenGL and OpenAL together but not SDL?

Reply:
You are correct.. done.